src/loans/solend.py

```diff
@@ -35,8 +38,10 @@
     # 'flash_repay_reserve_liquidity': '',
 }
 
+WAD = 10**18
```

Comment:
What does WAD stand for?

Reply:
it s very missleading abbreviation:
```

Yes, "WAD" in the context of blockchain and decentralized finance (DeFi) is an abbreviation. It stands for "Weighted Average Dividend," though in the context of Solidity and smart contracts, it is commonly understood as simply representing a fixed-point number with 18 decimal places, ensuring high precision in financial calculations.
```

Reply:
missleading abbreviation but in a nutshell:

> Yes, "WAD" in the context of blockchain and decentralized finance (DeFi) is an abbreviation. It stands for "Weighted Average Dividend," though in the context of Solidity and smart contracts, it is commonly understood as simply representing a fixed-point number with 18 decimal places, ensuring high precision in financial calculations.